### PR TITLE
state convert blog

### PR DESCRIPTION
**Pull request by Frassle**
## Description

Blog post about the new `import --from terraform` support.

## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

File: themes/default/content/blog/converting-full-terraform-states-to-pulumi/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Converting Full Terraform States to Pulumi"
+date: 2023-06-12
```
Comment (jaxxstorm) [inline themes/default/content/blog/converting-full-terraform-states-to-pulumi/index.md:3]: Incorrect date?

File: themes/default/content/blog/converting-full-terraform-states-to-pulumi/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+meta_desc: Learn how to convert whole Terraform states to Pulumi using the new Terraform converter
+meta_image: meta.png
+authors:
+    - fraser-waters
+tags:
+    - migration
+---
+
+Building on our work of [converting Terraform projects](/docs/converting-full-terraform-programs-to-pulumi), we now have support for converting Terraform state.
```
Comment (jaxxstorm) [inline themes/default/content/blog/converting-full-terraform-states-to-pulumi/index.md:12]:
```suggestion
Building on our work of [converting Terraform projects](/docs/converting-full-terraform-programs-to-pulumi), we now have support for adopting resources from Terraform state
```

**Review by jaxxstorm — APPROVED** [review]

File: themes/default/content/blog/converting-full-terraform-states-to-pulumi/index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+<!--more-->
+
+In our last blog post about converting from Terraform we called out that while it was great that the converter could migrate a Terraform project, but what if you had existing infrastructure resources.
+
+Historically we've shared some AWS specific code for [TypeScript](https://github.com/pulumi/tf2pulumi/blob/master/misc/import/import.ts) and [Go](https://github.com/pulumi/tf2pulumi/blob/master/misc/import-go/import.go) to import from `.tfstate` files. These have been useful, but they're limited to the AWS provider and either Go or TypeScript programs.
+
+From [v3.87.0](/docs/install) we have extended our converter system to state not just code. So now it's possible to import resources from other state system, the first of which we have support for being being Terraform again.
```
Comment (scottslowe) [inline themes/default/content/blog/converting-full-terraform-states-to-pulumi/index.md:20]:
```suggestion
From [v3.87.0](/docs/install) we have extended our converter system to also include state in addition to code. So now it's possible to import resources from other state systems, the first of which is Terraform.
```

**Review by scottslowe — CHANGES_REQUESTED** [review]

File: themes/default/content/blog/converting-full-terraform-states-to-pulumi/index.md (hunk 1 above)
Comment (Frassle) [reply themes/default/content/blog/converting-full-terraform-states-to-pulumi/index.md]: Yeh copy paste

**Comment by Frassle** [comment]
Need to update the meta.png for this

File: themes/default/content/blog/converting-full-terraform-states-to-pulumi/index.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    - fraser-waters
+tags:
+    - migration
+---
+
+Building on our work of [converting Terraform projects](/docs/converting-full-terraform-programs-to-pulumi), we now have support for adopting resources from Terraform state.
+
+<!--more-->
+
+In the prior blog post about converting from Terraform we called out that while it was great that the converter could migrate a Terraform project, but what if you had existing infrastructure resources.
```
Comment (scottslowe) [inline themes/default/content/blog/converting-full-terraform-states-to-pulumi/index.md:16]:
```suggestion
In the prior blog post about converting from Terraform we called out that it was great that the converter could migrate a Terraform project, but it didn't address situations where you had existing infrastructure resources.
```

**Review by scottslowe — APPROVED** [review]
One small suggested change (the wording seems awkward), but no need to block on it.

**Pull request merged**